app/pages/system/inventory/InventoryPage.tsx

```diff
@@ -28,6 +30,12 @@ export function InventoryPage() {
     <>
       <PageHeader>
         <PageTitle icon={<Racks24Icon />}>Inventory</PageTitle>
+        <ContextualDocsModal
+          heading="Sleds and Disks"
+          icon={<Racks24Icon />}
```

Comment:
We'll need a `<Racks16Icon />`, which I don't believe exists yet.

Reply:
@paryhin or @benjaminleonard, if one of you would be able to create a Racks16Icon, (or show me how,) I'd appreciate it!

Reply:
The 24 one was named wrong. It should be `Servers24` and you'll find `Servers16` already exists. I'll update the 24 one separately.